include/psa/crypto_types.h

```diff
@@ -380,4 +380,31 @@ typedef uint16_t psa_key_derivation_step_t;
 
 /**@}*/
 
+/** \defgroup pake Password-authenticated key exchange (PAKE)
+ * @{
+ */
+
+/** \brief Encoding of the side of PAKE
+ *
+ * Encodes which side of the algorithm is being executed. For more information
+ * see the documentation of individual PSA_PAKE_SIDE_XXX constants.
+ */
+typedef uint8_t psa_pake_side_t;
+
+/** Encoding of input and output indicators for PAKE.
+ *
+ * Some PAKE algorithms need to exchange more data than just a single key share.
+ * This type is for encoding additional input and output data for such
+ * algorithms.
+ */
+typedef uint8_t psa_pake_data_t;
```

Comment:
“Data” is vague. Should this type be called `psa_pake_step_t`? I'm not a great fan of “step”, but it's what we settled on for key derivation, and it's a very similar concept here.